File: binance4j-blvt/src/main/java/com/binance4j/blvt/dto/Basket.java
package com.binance4j.blvt.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * {@link Token} Basket.
 * 
 * @param symbol        Symbol.
 * @param amount        Amount.
 * @param notionalValue Notional value.
 */
@ApiModel("Token Basket.")
public record Basket(@ApiModelProperty("Symbol.") String symbol, @ApiModelProperty("Amount.") String amount,
		@ApiModelProperty("Notional value.") String notionalValue) {
}

File: binance4j-blvt/src/main/kotlin/com/binance4j/blvt/dto/Basket.kt
package com.binance4j.blvt.dto

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * [Token] Basket.
 * @property symbol        Symbol.
 * @property amount        Amount.
 * @property notionalValue Notional value.
 */
@ApiModel("Token Basket.")
data class Basket(
    @ApiModelProperty("Symbol.")
    var symbol: String? = null,
    @ApiModelProperty("Amount.")
    var amount: String? = null,
    @ApiModelProperty("Notional value.")
    var notionalValue: String? = null
)